### PR TITLE
Ported error_cleanup from autotest-client-tests to avocado

### PR DESCRIPTION
**Pull request by balamuruhans**
Have ported error_cleanup from autotest-client-tests to avocado

### CODE REVIEW

File: perf/error_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class error_cleanup(Test):
+    """
+    Raise an exception during cleanup()
+    """
+
+    def test(self):
+        pass
+
+    def cleanup(self):
```
Comment (apahim) [inline perf/error_cleanup.py:15]: In avocado we don't have `cleanup()`, but `tearDown()`.

**Comment by apahim** [comment]
Wondering what's the purpose of this test? Seems like a selftest for Avocado and if so, it should sit on avocado main repo, under `selftest/functional` directory.

**Comment by balamuruhans** [comment]
@apahim I tried to port simple test available from autotest to learn how avocado works. So, can change it with tearDown() and put it in selftest/functional or it is not required to be added?

**Comment by apahim** [comment]
@balamuruhans if you just change from `cleanup()` to `tearDown()` then you will have successfully ported the test. The `selftest/functional` part I mentioned is a different discussion, intended to understand the use and the relevance of the test. Also, since this is not a `perf` test for sure, you can create a new directory for it, like `generic`, as discussed here: https://github.com/avocado-framework/avocado-misc-tests/issues/8.

**Comment by balamuruhans** [comment]
@apahim thank you, I will add new directory as generic and add it.

**Pull request closed without merging**